STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerObservesTest.testObservesAnnotated
MODIFIED FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerObservesTest.java
CLEANER: de.strullerbaumann.visualee.examiner.cdi.TestGroup100Case0.test263
CLEANER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 19
NEW CLEANER SIZE: 2
ELAPSED TIME: 63.93300008773804
==========================
@@ -106,0 +106,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
